src/components/Common/Header.js

```diff
@@ -44,7 +37,7 @@ export default function Header() {
   const [showApprovedOnly, setShowApprovedOnly] = useState(false);
 
   const loadNotifcation = () => {
-    getNotification().then(res => {
+    getNotification().then((res) => {
```

Comment:
1. There is no async await.
2.No optional chaining.
3.

Reply:
resolved